docs/03. Planificación/Modelo de negocio.mdx

```diff
@@ -3,21 +3,31 @@
 | ------- | ---------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | 1.0     | 2024-04-27 | Elaboración inicial del documento para especificar nuestro modelo de negocio                                                                                 |
+| 1.1     | 2024-04-30 | Reestructuración del documento y cálculo del ROI                                                                                  |
 
 ## 1. Nuestro modelo de negocio
 
-El plan de negocio se centrará en desarrollar una PWA eficiente y fácil de usar que facilite la gestión de las operaciones de la ONG y las donaciones. Los ingresos para cubrir los costos de desarrollo y mantenimiento de la PWA provendrán de un 5% de cada donación realizada a través de la aplicación. Este modelo nos asegura que el beneficio que obtengamos será acorde a las donaciones recibidadas por las ONG, sin suponer un inconveniente, como podría ser el caso de establecer una tarifa fija por cada donación, que puede ser injusto para las donaciones más pequeñas. Con esto buscamos que las donaciones de una cantidad menor, que estimamos serán la mayoría, sean rentables. Para maximizar la eficiencia de costos, se ha llegado a la conclusión de que hay minimizar los costos de transacción con los procesadores de pagos, por lo que haremos uso de una pasarela de pago con Stripe.
-
-Es importante destacar que este trabajo se realiza sin ánimo de lucro, y cualquier excedente de ingresos después de cubrir los costos será reinvertido en la misión de la ONG.
+El plan de negocio se centrará en desarrollar una PWA eficiente y fácil de usar que facilite la gestión de las operaciones de la ONG y las donaciones. Los ingresos para cubrir los costos de desarrollo y mantenimiento de la PWA provendrán de un crecimiento lineal del número de ONGs clientes, lo que significa que los ingresos aumentarán de manera constante con el tiempo. En el primer año, se espera vender la aplicación a 5 ONGs, generando ingresos de 13.750 €. En los años siguientes, se espera que este número aumente de forma lineal, llegando a 20 ONGs en el cuarto año y generando un total de 143.500 € en ingresos durante este período. Para maximizar la eficiencia de costos, se ha llegado a la conclusión de que hay minimizar los costos de transacción con los procesadores de pagos, por lo que haremos uso de una pasarela de pago con Stripe.
 
 ### 1.1 Aspectos clave del modelo de negocio
 
-- Fuente de ingresos: Un 5% de cada donación realizada.
-
 - Propuesta de valor: Proporcionar una plataforma para facilitar la gestión de las operaciones de la ONG y las donaciones.
 
 - Actividades clave: Desarrollo y mantenimiento de la PWA, gestión de las donaciones, soporte al cliente.
```

Comment:
We do not manage donations

Reply:
I don't understand, we don't manage donations but I have deleted that line, what do you mean?

Reply:
![Captura de pantalla 2024-05-01 a las 18 37 04](https://github.com/ISPP-07/htld-doc/assets/80347035/bec38ce8-bfb8-49dd-b6ea-17f11edc3cc5)

Reply:
thank you, it's fixed now